src/bitmessagekivy/baseclass/network.py

```diff
@@ -17,7 +17,8 @@
     from pybitmessage.mock import kivy_main
     stats = kivy_main.network.stats
     objectracker = kivy_main.network.objectracker
-else:
+
+if state.py3_compatible:
```

Comment:
maybe `state.backend_py3_compatible`

Reply:
yes now i have forcefully pushed the changed and now it is same as suggested by you.